psalm-baseline.xml

```diff
@@ -857,6 +857,9 @@
     <ArgumentTypeCoercion>
       <code>$amount</code>
       <code>$amount</code>
+      <code><![CDATA[$fundingWithdrawal->getAmount()]]></code>
+      <code><![CDATA[$fundingWithdrawal->getAmount()]]></code>
+      <code><![CDATA[$fundingWithdrawal->getAmount()]]></code>
```

Comment:
These look like new issues but they are really existing issues being made more specific by adding more type info - `MixedAssignment` becomes `ArgumentTypeCoercion` when we unmix stuff.